src/fides/api/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -174,6 +175,7 @@ def privacy_experience_list(
             db=db, fides_user_device_id=fides_user_device_id
         )
 
+    await asyncio.sleep(delay=0.001)
```

Comment:
Interesting, where did you come up with this trick?

Reply:
I recall having to use this trick years ago but don't really remember where I learned it...probably from a book like `Fluent Python` if I had to guess